app/controllers/membership_applications_controller.rb

```diff
@@ -68,6 +68,11 @@ def information
 
   end
 
+  def destroy
+    @membership_application.destroy
+    redirect_to membership_applications_url, notice: t('membership_applications.application_deleted')
```

Comment:
You should use URLs with `redirect_to` : http://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to

The general guideline is:  `_url` in controllers (think: server side),` _path` in views (think: browser side)